lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -168,11 +168,11 @@ defmodule ExUnit.DocTest do
 
   Options can also be given:
 
-    * `:except` - generate tests for all functions except those listed
-      (list of `{function, arity}` tuples, and/or `:moduledoc`).
+    * `:except` - generates tests for all functions except those listed
+      (list of `{function, arity}` tuples, and/or `:moduledoc`)
 
-    * `:only`   - generate tests only for functions listed
-      (list of `{function, arity}` tuples, and/or `:moduledoc`).
```

Comment:
>  I haven't aligned any "-" in the PR, i don't know what are you referring to.

I'm betting José means that since you are changing these lines, remove the alignment since in the long run we want to remove it.

> I have realized that you use a trailing period, when you have more than one sentence, and you never finish a trailing period when it's only one sentence.

I didn't get this. But anyways, either use it for all bullet points or use it for no bullet point (I prefer the latter). :)